dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -139,10 +137,9 @@
     @BatchSize(size = 25)
     private SortedSet<User> users;
 
-    @ManyToMany(fetch = FetchType.EAGER)
+    @ManyToMany(fetch = FetchType.LAZY)
     @JoinTable(name = "starred", inverseJoinColumns = @JoinColumn(name = "userid", nullable = false, updatable = false, referencedColumnName = "id"), joinColumns = @JoinColumn(name = "entryid", nullable = false, updatable = false, referencedColumnName = "id"))
     @ApiModelProperty(value = "This indicates the users that have starred this entry, dockstore specific", required = false, position = 5)
```

Comment:
I think this needs to be removed and that you need an explicit `@JsonIgnore` --  we don't want it to ever come back in the response.